docs/advanced-topics/advanced-features.rst

```diff
@@ -326,14 +326,18 @@ with the `Cromwell <https://github.com/broadinstitute/cromwell>`__ version liste
 +-------------+-----------------------+
 |     1.9     |          44           |
 +-------------+-----------------------+
+|     1.10    |          44           |
++-------------+-----------------------+
+|     1.11    |          57           |
++-------------+-----------------------+
 
 
 Additionally, you can override the cromwell version in your
-``~/.dockstore/config`` using:
+``~/.dockstore/config`` using for example:
 
 ::
 
-    cromwell-version = 51
+    cromwell-version = 58
```

Comment:
I would pick a version that currently exists as an example, but I could see it this way too.